ImageAlertAction/Classes/UIAlertAction+Image.swift

```diff
@@ -11,6 +11,7 @@ import UIKit
 extension UIAlertAction {
 
     private var imageKey: String { "image" }
+    private var isSelectedKey: String { "checked" }
```

Comment:
Nice! Would you mind changing this to `isChecked`/`isCheckedKey` to mimic the key?
Especially since I can imagine wanting to check multiple actions, I think `check` vs `selected` is a better name.

Reply:
Makes sense, will change 👍

Reply:
Done ✅